config/config.yaml

```diff
@@ -84,6 +84,20 @@ environments:
     admin_creds: MasterOsAdminCredentials
     api_port: 8443
     # version: 3.6.1.44.16
+  ocp4:
```

Comment:
@openshift/team-qe fyi, now using ocp4 as BUSHSLICER_DEFAULT_ENVIRONMENT would let us need not add air_port and version for nextgen auto test